src/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -12,26 +13,20 @@ internal static class CookieExtensions
     {
         private static Func<Cookie, string> s_toServerStringFunc;
 
+        [PreserveDependency("ToServerString", "System.Net.Cookie", "System.Net.Primitives")]
```

Comment:
Did you try it without explicit tracking? The reflection call is easily detectable

Reply:
I didn't, though I also like the idea about being explicit about such reflection-based dependencies, not just for the linker, but to better help highlight such places via code searching and the like.

To be clear, though, you'd expect the linker to see this reflection usage and keep the target?

Reply:
> you'd expect the linker to see this reflection usage and keep the target

Yes

Reply:
Cool. I'll check.